resolver/request.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+
+import requests
+from validator_collection import checkers
+
+from sceptre.resolvers import Resolver
+
+
+def make_request(url):
```

Comment:
Why did you not make this a part of your resolver class? This seems like it belongs in your resolver.

Reply:
it has been moved into the class.